.ci/riscv-fuzz.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -e -u -o pipefail
+
+# compile
+command -v clang &> /dev/null
+
+make clean
+clang \
+    -g -O1 \
+    -fsanitize=fuzzer,address,undefined \
+    -include src/common.h \
+    -D RV32_FEATURE_EXT_F=0 \
+    -D RV32_FEATURE_SDL=0 \
+    -D DEFAULT_STACK_ADDR=0xFFFFE000 \
+    -D DEFAULT_ARGS_ADDR=0xFFFFF000 \
+    -D FUZZER \
+    -o build/rv32emu_fuzz \
+    src/fuzz_target.cc \
+    src/map.c \
+	src/utils.c \
+	src/decode.c \
+	src/io.c \
+	src/syscall.c \
+	src/emulate.c \
+	src/riscv.c \
+	src/elf.c \
+	src/cache.c \
+	src/mpool.c \
+	src/main.c
+
+# populate the initial CORPUS for the fuzzer using valid elf
+mkdir -p build/fuzz/CORPUS_DIR
+cp build/*.elf build/fuzz/CORPUS_DIR
```

Comment:
Don't copy all ELF files. Instead, you shall select some of them, which have been validated.

Reply:
May I ask what you mean by "validated"?